independent-projects/bootstrap/app-model/src/test/java/io/quarkus/util/GlobUtilTest.java

```diff
@@ -34,6 +34,8 @@ void doubleStar() {
         assertMatch("a**/b", Arrays.asList("a/b", "axy/b", "a/x/b"), Arrays.asList("a", "b", "a/bc", "bc/b"));
         assertMatch("a**b", Arrays.asList("ab", "axb", "axyb", "a/b", "a/x/b"), Arrays.asList("abc", "1ab"));
         assertMatch("a**b**/c", Arrays.asList("axbx/c", "axbx/c", "a/x/xbx/c", "axbx/xxx/c"), Arrays.asList("axbx/cc"));
+        assertMatch("**/*.txt", Arrays.asList("/test.txt", "test.txt", "/path/to/a.txt", "relative/path/to/a.txt"),
+                Arrays.asList("/test.py", "test.json", "/path/to/a.js", "relative/path/to/a.exe"));
```

Comment:
Could you add a test with `foo/**/test.json` and `foo/**/*.json`?

Reply:
Done